Update swagger file for live data

tejaswinichile opened the pull request.

ghidinelli committed:
Fix missing type def for the array of notes

chrisredrich committed:
Updated the passings model to include missing fields

chrisredrich committed:
revert accidental commit to master

tejaswinichile committed:
Update swagger file for live data

chrisredrich commented on swagger.yaml:
```diff
@@ -288,6 +288,73 @@ paths:
         404: 
           description: The requested transponder is not currently live
+          
+  '/events/{event_slug}/live/leaderboard.html':
```
This endpoint supports both a json and rss response. We need to add a response format for each as well as how to request the rss response instead of json.

chrisredrich commented on swagger.yaml:
```diff
@@ -288,6 +288,73 @@ paths:
+        - text/csv
+      summary: Retrieve live run data in CSV format
+      parameters:
+        - name: event_slug
```
These should be `event_id` throughout. We don't use `event_slug`.

chrisredrich commented on swagger.yaml:
```diff
@@ -288,6 +288,73 @@ paths:
+        '429':
+          $ref: '#/responses/RetryLaterError'
+
+  '/events/{event_slug}/live/racers.csv':
```
The summary describes this as format of rss, but the request url shows csv.

chrisredrich reviewed (changes requested):
@tejaswinichile I left some comments throughout. Additionally a few things:
1. Can you update this pull request to commit to the `scoring-systems-api` branch?
2. It seems we are missing documentation for the api.racehero.io/v1/events/:id/live/results.rss endpoint. 
3. I'm not sure if you branched from Master for this, but this version seems to be missing a lot of the endpoints we documented in the `scoring-systems-api` branch.

tejaswinichile commented:
@chrisredrich I have added the Enum please have a look.

Commit:
Updated swagger document, according to PR comments

1. Added missing results.rss request
2. added leaderboard.rss missing response
3. updated format for racers

tejaswinichile committed:
Merge branch 'scoring-systems-api' into patch-2

tejaswinichile commented:
@chrisredrich I have updated the document according to the PR comments and also rebased it with the `scoring-systems-api` branch.

sophiascto reviewed (commented):
https://github.com/RaceHero/api/pull/4#issue-415107066
